test/benches/bench.rs

```diff
@@ -276,8 +276,8 @@ criterion_group!(
     benches,
     wasm_module_compile,
     wasm_module_deserialize_from_file,
-    wasm_module,
-    wasm_instance,
+    // wasm_module,
+    // wasm_instance,
```

Comment:
currently the bench fails because such numerous deserialization of modules causes memory leaks because of an upstream issue where the memory for deserialization is kept as long as the engine lives https://github.com/wasmerio/wasmer/issues/4377#issuecomment-1879386384

Reply:
Can a note be left in the code here so we don't have to trace back to this PR to figure out why these are commented out later?